src/lib/viewers/BaseViewer.js

```diff
@@ -545,6 +545,7 @@ class BaseViewer extends EventEmitter {
      */
     handleFullscreenEnter() {
         this.resize();
+        this.emit(ANNOTATOR_EVENT.toggleVisibility);
```

Comment:
Should this be `this.annotator.emit`, instead? If not, how does it get into the annotator?

Reply:
> Should this be `this.annotator.emit`, instead? If not, how does it get into the annotator?

😱